contentcuration/contentcuration/views/internal.py

```diff
@@ -654,6 +682,7 @@ def create_node(node_data, parent_node, sort_order):  # noqa: C901
         freeze_authoring_data=True,
         role_visibility=node_data.get('role') or roles.LEARNER,
         complete=is_complete,
+        **metadata_labels
```

Comment:
Should we be more explicit here by providing kwargs using `metadata_labels`?

Reply:
The intention here is to only set the kwargs for each of the metadata_labels keys if they have been set, so I don't think there's another way to do that except to explicitly write `<kwarg>=metadata_labels.get("<kwarg>")`